server/controllers/userToMentor.js

```diff
@@ -1,50 +1,48 @@
-import  bcrypt from 'bcrypt';
-import  users from '../models/users';
+
+import pool from '../configurations/db-config';
 
 const changeUserToMentor = async(req,res)=>{
-    const checking = users.find((objectof) => objectof.id === parseInt(req.params.userId));
+    try{
+        const mentors ='SELECT * FROM USERS'
```

Comment:
Same as the comment of the previous PR, It's efficient to only fetch a user with the specified id rather than looping through all users.